core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -334,7 +349,7 @@ angular.module('oppia').directive('storyNodeEditor', [
             // EXPLORATION_AND_SKILL_ID_PATTERN
             // is not being used here, as the chapter of the story can be saved
             // with empty exploration id.
-            $scope.explorationIdPattern = /^[a-zA-Z0-9_-]*$/;
+            $scope.explorationIdPattern = /^[a-zA-Z0-9_-]+$/;
```

Comment:
With this change, what does the flow look like if the exploration ID field is empty?

Reply:
The save button is disabled when the exploration ID field is empty.